### PR TITLE
[MIG] product_template_multi_link: Migration to 14.0

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

**Review by dsolanki-initos — APPROVED** [review]
Code review and Functional Tests. LGTM

File: product_template_multi_link/README.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (simahawk) [inline product_template_multi_link/README.rst]: this README is outdated. Can you move to readme fragments?

**Review by claudiagn — APPROVED** [review]
functional review, LGTM!

**Comment by dsolanki-initos** [comment]
It seems that PR is ready to merge

**Comment by dsolanki-initos** [comment]
Can this be merged?

**Review by chienandalu — CHANGES_REQUESTED** [review]
https://github.com/OCA/e-commerce/pull/489#discussion_r598594845 wasn't attended. Please do

**Comment by dsolanki-initos** [comment]
Hi @chafique-delli can you please resolve above comments so that we can merge this PR?

File: product_template_multi_link/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Sylvain LE GAL <http://www.twitter.com/legalsylvain>
```
Comment (simahawk) [inline product_template_multi_link/readme/CONTRIBUTORS.rst:1]: * Laurent Mignon <laurent.mignon@acsone.eu>
* Simone Orsi <simahawk@gmail.com>

**Review by simahawk — APPROVED** [review]

**Comment by dsolanki-initos** [comment]
It would be great if we could merge this PR.

**Comment by chienandalu** [comment]
Could you attend the last comment, please?

**Comment by dsolanki-initos** [comment]
Hi @chafique-delli can you please have a look at the comments so that we can merge the PR ASAP.

**Comment by dsolanki-initos** [comment]
Hi @chafique-delli It would be great if you could attend the comments

**Comment by dsolanki-initos** [comment]
It would be great if you could attend the comments so that we can proceed further with the PR.

**Pull request merged**